### ISSUE TITLE
A way to set up a backup slot

### ISSUE THREAD

**Issue by jcoffland** [FoldingAtHome/fah-client-pub#81]
**Reported by @Ivoshiee on 7 Jul 2010 10:55 UTC**
It would be useful to have a some sort of "backup slot" capability in FAH client to address assignment issues or equipment issues (like GPU, driver problems etc). It is to address cases when there are no WUs for given slot type, these WUs can not be assigned at the moment or some run-time client side error detection determines that some slot is (temporarily) not available for computation. It has to be a run-time slot regeneration and require no restarting of the client. 
It is natural to assume that SMP slot can be backed up by number of single proc slots or GPU slot can be backed up a single proc slot etc.
Swithing between backup slot and a primary slot should be performed only after a backup slot is finished a WU.

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 15 Jul 2010 19:48 UTC**
This is a reasonable idea but it would also introduce a lot of complication.  Thus the low priority.

**Comment by jcoffland** [comment]
**Comment by @bb30994 on 26 Jul 2010 02:53 UTC**
Yes, it's very complicated.

Here's a similar scenario.  Currently I'm running [3 plus one classic](-smp).  I want to set the oneunit flag for the classic (which can now be done easily in the GUI) and when it finishes, change -smp 3 to -smp 4.  Good luck with that one. :D
